Formula/tbb.rb

```diff
@@ -17,6 +17,11 @@ class Tbb < Formula
   depends_on "swig" => :build
   depends_on "python@3.8"
 
+  patch do
+    url "https://github.com/oneapi-src/oneTBB/pull/258/commits/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.diff"
```

Comment:
Please use the repository URL, not a pull request URL.

Reply:
https://github.com/intel/tbb redirects to https://github.com/oneapi-src/oneTBB

Reply:
I mean that this should use `https://github.com/oneapi-src/oneTBB/commit/86f6dcdc17a8f5ef2382faaef860cfa5243984fe.diff?full_index=1` rather than the pull request URL, which are not stable and don't support the `full_index` parameter.

Reply:
@jonchang Thanks for the feedback, I wasn't even aware these kind of urls were available (everything linked in the PR refers to a `/pull` url so this is a very useful tip - thanks!